### ISSUE TITLE
Attention mask definition.

### ISSUE THREAD

**Issue by Chuck-Yu** [cnzzx/GeMap#1]
Hi, thanks for your great work. 

After reading your paper and code, I feel confuse about the attn_mask definition. According to [MultiheadAttention.attn_mask](https://pytorch.org/docs/stable/generated/torch.nn.MultiheadAttention.html), 'For a binary mask, a _True_ value indicates that the corresponding position is not allowed to attend.' 

https://github.com/cnzzx/GeMap/blob/b2b085e1584ca767c9d384e51b96f209babb7234/projects/mmdet3d_plugin/gemap/modules/geometry_decoupled_attention.py#L74-L78

**Comment by cnzzx** [comment]
Thanks for your issue. You're right.

There might be some mistakes when cleaning up the code. We've now fixed it up and you can view it in [this commit](https://github.com/cnzzx/GeMap/commit/fba8f084830088ace4f8817870682d4f92924ae0).
